dbt/include/snowflake/macros/materializations/incremental.sql

```diff
@@ -11,20 +10,41 @@
        for (presumably) faster overall incremental processing.
 
        Low-level specifics:
+       If an invalid option is specified, then we will raise an
+       excpetion with corresponding message.
+
        Languages other than SQL (like Python) will use a temporary table.
        With the default strategy of merge, the user may choose between a temporary
        table and view (defaulting to view).
-       The append strategy can use a view because it will run a single INSERT statement.
-       When the unique_key is none,
-       then we can use a view because it will run a single INSERT statement.
-       Otherwise, play it safe by using a temporary table.
+
+       The append strategy can use a view because it will run a single INSERT statement
+       when the unique_key is none.
```

Comment:
It looks like "when the unique_key is none" got accidentally duplicated and applied to the `append` strategy. I think it was only intended to apply to the `delete+insert` strategy.
```suggestion
```

Reply:
the append strategy part actually was already there I believe can remove it though if it seems unnecessary.

Reply:
Yeah, I think we'll want to remove it because it's actually inaccurate.